Makefile

```diff
@@ -9,6 +9,10 @@ GO       := GO111MODULE=on go
 GOBUILD  := CGO_ENABLED=0 $(GO) build
 GOTEST   := CGO_ENABLED=1 $(GO) test
 PACKAGES  := $$(go list ./... | grep -vE 'tests|cmd|vendor|pbmock')
+PACKAGES_RELAY := $$(go list ./... | grep -vE 'tests|cmd|vendor|pbmock' | grep 'github.com/pingcap/dm/relay')
+PACKAGES_SYNCER := $$(go list ./... | grep -vE 'tests|cmd|vendor|pbmock' | grep 'github.com/pingcap/dm/syncer')
+PACKAGES_PKG_BINLOG := $$(go list ./... | grep -vE 'tests|cmd|vendor|pbmock' | grep 'github.com/pingcap/dm/pkg/binlog')
```

Comment:
These three expressions don't need the `grep -vE 'tests|cmd|vendor|pbmock'` filter

Reply:
based on `PACKAGES`?

Reply:
like this
```
PACKAGES_RELAY := $$(go list ./... | grep 'github.com/pingcap/dm/relay')
```

Reply:
Wo, I'm so stupid. 😭

Reply:
addressed in https://github.com/pingcap/dm/pull/416/commits/078b38c5a3258eeada6d36ae0f8032e3634e62f6